front/lib-svelte/src/ui/Icone.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  type Icone = 'check' | 'fleche-gauche';
+  export let type: Icone;
```

Comment:
Je me dis qu'il faudrait peut être mieux ne pas typer avec Icone. Laisser plutôt un nom d'icone correspondant à la fin de son url. Et aussi standardiser le nom des icones pour les intégrer plus facilement. 

```suggestion
  export let nom: string;
```

Permet de ne plus avoir à spécifier le path pour chaque icone et rendre ça plus facile à l'implémentation et avoir quelques choses comme ce qui suit.

```
  @each $name {
    .icone.#{$name} {
      mask-image: url(/assets/icones/#{$name});
      -webkit-mask-image: url(/assets/icones/#{$name});
    }
  }
```

Reply:
Ça fonctionne `@each $name` ?